src/signals/incident-management/index.js

```diff
@@ -106,10 +111,10 @@ const IncidentManagement = () => {
             <Route path={routes.defaultTexts} element={<DefaultTextsAdmin />} />
           )}
           {configuration.featureFlags.showStandardTextAdminV2 && (
-            <Route
-              path={routes.standardTexts}
-              element={<StandardTextsAdmin />}
-            />
+            <Route path={routes.standardTexts}>
+              <Route index element={<StandardTextsAdmin />} />
+              <Route path={`:id`} element={<StandardTextsDetail />} />
+            </Route>
```

Comment:
Not sure if this is the correct way to implement this. @OscarBakker it seems with IncidentDetail and IncidentOverview it works differently. Can you check this?

Reply:
@OscarBakker If it is not the correct way, how do I then do the BackLink? Cause I ran into trouble there and this was my solution. (all right stackoverflow's solution)